tests/benchmarks/timing_info/prj_userspace.conf

```diff
@@ -0,0 +1,10 @@
+CONFIG_TEST=y
+CONFIG_EXECUTION_BENCHMARKING=y
+CONFIG_TEST_RANDOM_GENERATOR=y
+CONFIG_HEAP_MEM_POOL_SIZE=256
+CONFIG_MAIN_STACK_SIZE=2048
```

Comment:
why is main stacksize increased?

Reply:
actaully no specific reason. These configs are taken from the old prj.conf file.